### DISCUSSION TITLE
Failed to run vertical logistic federated learning on multiple machines

### DISCUSSION THREAD

**Discussion by relinda2019** [alibaba/FederatedScope#230] [Q&A] [answered]
When run FederatedScope\federatedscope\vertical_fl\worker\vertical_server.py ---> trigger_for_start function -->self.broadcast_client_address() , The following problem will occur.Has anyone encountered the same problem? Can you tell me how to solve it. Many thanks!!
**Error**:  "TypeError: 1 has type int, but expected one of: bytes, unicode"

![20220713-200037](https://user-images.githubusercontent.com/39260479/178729011-27401a7e-f44a-4faf-9469-3582ad2562b5.jpeg)

**Comment by xieyxclack** [comment, marked as answer]
Thanks for your comments. We have fixed the issue in #239 , which supports `int` type of keys.
